crud/tarantool_test.go

```diff
@@ -87,6 +87,11 @@ var opObjManyOpts = crud.OperationObjectManyOpts{
 	Timeout: crud.MakeOptFloat64(timeout),
 }
 
+var schemaOpts = crud.SchemaOpts{
+	Timeout: crud.MakeOptFloat64(timeout),
+	Cached:  crud.MakeOptBool(false),
```

Comment:
```suggestion
	Timeout: crud.MakeOptFloat64(timeout),
	VshardRouter: crud.MakeOptString("foo"),
	Cached:  crud.MakeOptBool(false),
```

Reply:
There are integration tests which will start to fail due to unknown Cartridge vshard group name

Reply:
Preparing proper tests for vshard group is rather painful (it seems to be the reason why other requests also don't test it now).

Reply:
You could make a separate SchemaOpts for the request unit-tests. I'm ok not to test it in an integration test.

Reply:
Added a separate unit test suite in a separate commit.

(I had also discovered that I had forgot to actually encode opts, now it's fixed.)